components/fxa-client/src/oauth.rs

```diff
@@ -259,25 +259,45 @@ impl FirefoxAccount {
     }
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize)]
+#[derive(Clone, Serialize, Deserialize)]
 pub struct RefreshToken {
     pub token: String,
     pub scopes: HashSet<String>,
 }
 
+impl std::fmt::Debug for RefreshToken {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
```

Comment:
nit: `fmt.debug_struct("RefreshToken").field("scopes", &self.scopes).finish()`

here and elsewhere.

Reply:
Thank you! I figured there must be a way to do that but didn't know the right google-fu to figure it out.